File: Aula 04 - Estruturas de Decisao/cpp_logic/desvio_simples.cpp
#include <iostream>
using namespace std;

int main() {
    int quantidade;
    cout << "Digite quantidade em estoque: ";
    cin >> quantidade;
    
    // Usando a estrutura if 
    if (quantidade < 5) {
        cout << "AVISO: O estoque desse produto está baixo!" << endl;
    } 

    return 0;
}

File: Aula 04 - Estruturas de Decisao/java_logic/desvio_simples.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner scanner = new Scanner (System.in);
        System.out.print("Digite a quantidade em estoque: ");
        int quantidade = scanner.nextInt();

        // Usando a estrutura if
        if (quantidade < 5) {
            System.out.println("AVISO: O estoque desse produto está baixo!");
        }
        scanner.close();
    }
}